myhoard/controller.py

```diff
@@ -1389,6 +1389,11 @@ def _send_binlog_stats(self):
         self.stats.gauge_float("myhoard.binlog_upload_age_min", min(binlog_upload_ages))
         self.stats.gauge_float("myhoard.binlog_upload_age_max", max(binlog_upload_ages))
 
+        # Send additional info about how long 'complete' binlog files have been waiting to be uploaded
+        binlog_upload_delays = [backup_stream.binlog_upload_delay for backup_stream in backup_streams]
+        self.stats.gauge_float("myhoard.binlog_upload_delay_min", min(binlog_upload_delays))
```

Comment:
Minor nitpicking, sorry, I hadn't seen this before.

You're using gauge_float but you seem to return an int from  binlog_upload_delay, so, the types don't match. You should choose the proper gauge_ function.

Could you please also add the return type annotation in the binlog func?